e2e_tests/support/utils.js

```diff
@@ -3,7 +3,7 @@
 export const isEnterpriseEdition = () =>
   Cypress.config('serverEdition') === 'enterprise'
 
-export const isAura = () => Cypress.config('serverEdition') === 'aura'
+export const isAura = () => Cypress.config('serverEdition').startsWith('aura')
```

Comment:
I guess this should do a conversion to lower case before the startsWith is called, since we are currently migration away from the lower case `aura` to the uppercase `AuraProfessional` and `AuraEnterprise`

Reply:
This is comparing against an internal browser value, which is `aura` or `auraenterprise`

Reply:
ah ok sorry, then forget my comment 🙏 

Thank you 🎉